### DISCUSSION TITLE
2015/12/23/some-screenshots-of-owarimonogatari/index

### DISCUSSION THREAD

**Discussion by giscus** [sumy7/sumy7.github.io#8] [Announcements]
# 2015/12/23/some-screenshots-of-owarimonogatari/index

SumyGG's Blog

https://sumygg.com/2015/12/23/some-screenshots-of-owarimonogatari/index.html

**Comment by aojunhao123** [comment]
哭死，我也是物语系列死忠粉

**Reply by sumy7** [reply]
握爪~
